### PR TITLE
Feature Create frontend UI for electricity calculator

### LINKED ISSUE #43: As a user, I want to be able to calculate my electrical consumption footprint
**__User Story Description__**
As a user
I want to I want to be able to calculate my electrical consumption footprint
So I can see my CO2 emissions results

**__Steps to Follow (optional)__**
- [x] Create form on '/electricity' page
- [x] Communicate with Developer on Issue #20 for questions to put in UI
- [x] Form on submit should send correct data to backend
- [x] Additional steps as necessary

**__Additional Considerations__**
Keep in mind the backend version of this ticket #20

### PR DESCRIPTION
**Pull request by TiciaD**
- added type for CountryCode in Electricty schema
    - Created Response and Form components for Electricity pages
    - Created json file to contain states and provinces for dropdown
    - Added states.json file containing states and provinces data
    - renamed watering plants image to remove spaces from filename

Closes #43

### CODE REVIEW

File: src/components/Electricity/ElectricityForm.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  };
+
+  const resetForm = () => {
+    setCountry(() => 'us');
+    setState(() => 'al');
+    setElectricityValue(() => 0);
+    setElectricityUnit(() => 'mwh');
+  };
+
+  console.log(
```
Comment (ialej001) [inline src/components/Electricity/ElectricityForm.tsx]: Should we not check in debugging console logs into dev/main?

File: src/pages/electricity.tsx [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
+  const { mutate, data, isLoading } = trpc.useMutation([
+    'electricity.unregistered-request-electricity',
+  ]);
+  const handleSubmit = (electricityData: UnregisteredElectricityRequest) => {
+    mutate({ ...electricityData });
+  };
+
+  if (isLoading) {
+    return (
+      <div className="mx-auto">
```
Comment (ialej001) [inline src/pages/electricity.tsx]: When I ran this locally, the spinner was not centered. Maybe change the classes to 'flex items-center mt-5' just show it centers on the page with a bit of margin on the top?

**Review by ialej001 — APPROVED** [review]
The styling on the spinner is a non-issue, just a suggestion. I do think we shouldn't check in any console logs however.

Otherwise, excellent work.

File: src/components/Electricity/ElectricityForm.tsx (hunk 1 above)
Comment (TiciaD) [reply src/components/Electricity/ElectricityForm.tsx]: Nice catch! Updated!

File: src/pages/electricity.tsx (hunk 2 above)
Comment (TiciaD) [reply src/pages/electricity.tsx]: Yeah it did look weird, decided to move it after where the picture is rendered and added `justify-center` so hopefully looks a little better!

**Review by ialej001 — APPROVED** [review]

**Review by victoriacheng15 — COMMENTED** [review]
I think the electricity input should not go below zero? 
image

Beside the above, everything else looks good.

**Review by amandaalexandre — APPROVED** [review]

**Comment by TiciaD** [comment]
> I think the electricity input should not go below zero? image
> 
> Beside the above, everything else looks good.

Good catch Victoria! Just updated it!

**Pull request merged**